src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java

```diff
@@ -81,6 +82,10 @@
      * Topic name.
      */
     private transient String topic;
+    /**
+     * Hashtags.
+     */
+    private transient List<String> hashtags;
```

Comment:
From what I can see this should not be `transient`.

```suggestion
    private List<String> hashtags;
```

Reply:
Done